mgmt/src/processor/proc.rs

```diff
@@ -349,13 +352,31 @@ async fn apply_router_config(
     Ok(())
 }
 
+/// refresh mappings for per vpc statistics
+fn update_stats_vpc_mappings(config: &GwConfig, vpcmapw: &mut VpcMapWriter<VpcMapName>) {
+    // create a mapping table frome the vpc table in the config
+    // FIXME(fredi): visibility
+    // FIXME(fredi): generalize the vpcmapName table
+    let vpc_table = &config.external.overlay.vpc_table;
+    let mut vpcmap = VpcMap::<VpcMapName>::new();
+    for vpc in vpc_table.values() {
+        let disc = VpcDiscriminant::VNI(vpc.vni);
+        let map = VpcMapName::new(disc, &vpc.name);
```

Comment:
Does this clear the stats on every reconfiguration?  If so, is there any way we can not do this?  I will not have this block this PR, but if it does clear stats for an existing VPC, can we create an issue to fix that?

Reply:
No, this does no clear the stats. This just reconfigures the VPC discriminant-to-VPCname mappings. In fact, the opposite should be addressed. If at some point we're measuring stats for some VPC "A" and, later, that VPC "A" no longer exists because it's removed from the config, the stats for it should be removed and we don't. I will address this one, which I forgot.